social_django/fields.py

```diff
@@ -41,8 +41,12 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault('default', dict)
         super(JSONField, self).__init__(*args, **kwargs)
 
-    def from_db_value(self, value, expression, connection, context):
-        return self.to_python(value)
+    if django.VERSION < (2,):
+        def from_db_value(self, value, expression, connection, context):
+            return self.to_python(value)
+    else:
+        def from_db_value(self, value, expression, connection):
+            return self.to_python(value)
```

Comment:
Why not something simpler like:

```
def from_db_value(self, value, expression, connection, *extra):
    return self.to_python(value)
```
?

Reply:
Yeah, makes sense.
Ameded.  I've tried to use a var name that explains what it is for.

Reply:
Well, it will not work if `context` is used as kwarg.  Should we also have `**extra` then?

Reply:
Why not just fetch all you don't need like:
```python
def from_db_value(self, value, *args, **kwargs):
    return self.to_python(value)
```

Reply:
That would make the method somewhat future prove ;)

Reply:
Makes sense, amended again.